modules/gitops-configuring-argo-cd-using-dex-github-conector.adoc

```diff
@@ -0,0 +1,33 @@
+[id="configuring-argo-cd-using-dex-github-connector_{context}"]
+= Configuring Argo CD SSO using Dex GitHub connector
```

Comment:
Please remove the whole section as OpenShift-GitOps just supports the OpenShift connector and nothing else. GitHub connector is supported by the upstream operator but we dont have to mention it in the OpenShift docs.